javav2/example_code/cloudwatch/Readme.md

```diff
@@ -1,90 +1,98 @@
+### Prerequisites
 
-Define these values to successfully run the JUnit tests:
+To run these examples, set up your development environment. For more information, 
+see [Get started with the SDK for Java](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/setup.html). 
 
-- **logGroup** - The name of the log group to use. For example, **testgroup**.
+ To run these examples, you require three JSON files. You can find these three files in this Github repository. The name of these files are: jsonWidgets.json, CloudDashboard.json, and settings.json. The  CloudWatchScenario depends on these files. 
+ 
+ **Be very careful** when running an operation that deletes or modifies AWS resources in your account. We recommend creating separate test-only resources when experimenting with these examples.
 
-- **alarmName** – The name of the alarm to use. For example, **AlarmFeb**.
+ ## Test the examples
+ 
+ ⚠️ Running the tests might result in charges to your AWS account.
 
-- **instanceId** – The ID of the instance to use. You can obtain this value from the AWS Management Console. For example, **ami-04300000000**.
+You can test the Java code example for Amazon Keyspaces by running a test file named **KeyspaceTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/java** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
 
-- **streamName** - The name of the stream to use. This value is used to retrieve log events.
+To successfully run the JUnit tests, define the following values in the test:
 
+- **logGroup** - The name of the log group to use. For example, **testgroup**.
+- **alarmName** – The name of the alarm to use. For example, **AlarmFeb**.
+- **instanceId** – The ID of the instance to use. You can obtain this value from the AWS Management Console. For example, **ami-04300000000**.
+- **streamName** - The name of the stream to use. This value is used to retrieve log events.
 - **ruleResource** – The Amazon Resource Name (ARN) of the user who owns the rule. You can obtain this value from the AWS Management Console.  
 -  **filterName**  - The name of the filter to use.
-
 - **destinationArn** - The ARN of the destination. This value is used to create subscription filters.
-
 - **roleArn** - The ARN of the user. This value is used to create subscription filters.
-
 - **filterPattern** - The filter pattern. For example, **Error**.
+- **myDateSc** - The start date to use to get metric statistics in the scenario test. (for example, 2023-01-11T18:35:24.00Z). 
+- **costDateWeekSc** - The start date to use to get AWS/Billinget statistics. (for example, 2023-01-11T18:35:24.00Z). 
```

Comment:
- **costDateWeekSc** - The start date to use to get AWS billing statistics. (For example, 2023-01-11T18:35:24.00Z.)